tools/generators/legacy/src/Generator/AddTargets.swift

```diff
@@ -192,27 +192,17 @@ Product for target "\(key)" not found in `products`
             return nil
         }
 
-        let copyOutputs: String
-        if let bazelBasename = outputs.productBasename {
-            copyOutputs = #"""
-else
-  "$BAZEL_INTEGRATION_DIR/copy_outputs.sh" \
-    "\#(Generator.bazelForcedSwiftCompilePath)" \
-    "\#(bazelBasename)" \
-    "\#(productType.rsyncExcludeFile)"
-"""#
-        } else {
-            copyOutputs = ""
-        }
-
```

Comment:
We still need this conditional. I think instead of the full remove like I suggested (sorry) we can change `outputs.productBasename` to `outputs.hasProductOutput` (similar to the `hasSwiftOutputs`.

Reply:
I have a question though: isn't checking `output.productBasename` the same as checking whether `BAZEL_OUTPUTS_PRODUCT` is set? I guess there's a subtle difference but I couldn't find any.

Reply:
Hmm. Yeah. I guess we can unconditionally run the script. It was probably an optimization we don't need.

Reply:
Ok, then this change should be good to go. Are any other changes needed?

Reply:
Not that I know of. I need about 15 mins before I can start to review this again. It’s probably going to be merged as is though.